topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -68,6 +68,12 @@ def populate_item(self, metadata_row, asset: Asset = None) -> None:
 
         if asset:
             item.add_asset(asset)
+        else:
+            # Required in the historical-imagery STAC extension
+            # and not added in case of 'validate' because not based on 'tiff' files
+            item.properties["proj:epsg"] = None
```

Comment:
I think this should be eo:bands rather than proj:epsg, but since there is no asset (as this is just a metadata validator) and it's required by the historical imagery schema, that does raise a few questions - we should discuss with @dwsilk